src/modules/fw_mode_manager/FixedWingModeManager.cpp

```diff
@@ -1263,6 +1263,75 @@ FixedWingModeManager::control_auto_takeoff(const hrt_abstime &now, const float c
 	}
 }
 
+void
+FixedWingModeManager::control_auto_takeoff_no_nav(const hrt_abstime &now, const float control_interval,
+		const float current_setpoint_altitude_amsl)
+{
+	if (!_control_mode.flag_armed) {
+		reset_takeoff_state();
+	}
+
+	const float takeoff_airspeed = (_param_fw_tko_airspd.get() > FLT_EPSILON) ? _param_fw_tko_airspd.get() :
+				       _param_fw_airspd_min.get();
+
+	if (_runway_takeoff.runwayTakeoffEnabled()) {
+		// TODO runway takeoff without nav
```

Comment:
@sfuhrer Should we for now set throttle to 0 to make sure the vehicle does not do anything crazy? If not, then we need to at least test that it's safe.

Reply:
I'll address this today.